libuuu/bmap.cpp

```diff
@@ -1,3 +1,34 @@
+/*
+ * Copyright 2025 NXP.
+ *
```

Comment:
Done't claim nxp copyright for this file, which is added by no-nxp contribtor.

Reply:
Dropped